aspnetcore/host-and-deploy/scaling-aspnet-apps/scaling-aspnet-apps.md

```diff
@@ -0,0 +1,320 @@
+
+    In the steps ahead, you'll resolve this issue by centralizing the data protection keys in an Azure storage service and protecting them with Key Vault.
+
+## Create the Azure Services
+
+To resolve the errors impacting the container app, you'll create the following services and connect them to your app:
+
+* [Azure Storage Account](/azure/storage/common/storage-account-overview): Handles storing data for the Data Protection Services. Provides a centralized location to store key data as the app scales. Storage accounts can also be used to hold documents, queue data, file shares, and almost any type of blob data.
+* [Azure KeyVault](/azure/key-vault/general/basic-concepts): This service stores secrets for your app, and is used to help manage encryption concerns for the Data Protection Services.
+
+#### Create the storage account service
+
+1. In the Azure portal search bar, enter `Storage accounts` and select the matching result.
+1. On the storage accounts listing page, select **+ Create**.
+1. On the **Basics** tab, enter the following values:
+    * **Subscription**: Select the same subscription that you chose for the container app.
+    * **Resource Group**: Select the *msdocs-scalable-razor* resource group you created previously.
+    * **Storage account name**: Name the account *scalablerazorstorageXXXX* where the X's are random numbers of your choosing. This name must be unique across all of Azure.
+    * **Region**: Select a region that is close to you.
+1. Leave the rest of the values at their default and select **Review**. After Azure validates your inputs, select **Create**.
+
+Azure provisions the new storage account. When the task completes, choose **Go to resource** to view the new service.
+
+#### Create the storage container
+
+You'll need to create a Container to store your app's data protection keys.
+
+1. On the overview page for the new storage account, select **Storage browser** on the left navigation.
+1. Select **Blob containers**.
+1. Select **+ Add container** to open the **New container** flyout menu.
+1. Enter a name of *scalablerazorkeys*, leave the rest of the settings at their defaults, and then choose **Create**.
+
+You should see the new containers appear on the page list.
+
+#### Create the key vault service
+
+Create a key vault to hold the keys that protect the data in the blob storage container.
+
+1. In the Azure portal search bar, enter `Key Vault` and select the matching result.
+1. On the key vault listing page, select **+ Create**.
+1. On the **Basics** tab, enter the following values:
+    * **Subscription**: Select the same subscription that chose for your Container App.
+    * **Resource Group**: Select the *msdocs-scalable-razor* resource group you created previously.
+    * **Key Vault name**: Enter the name *scalablerazorvaultXXXX*.
+    * **Region**: Select a region near your location.
```

Comment:
same as previous region?

Reply:
I thought this might get confusing depending on which service they're creating and which are available in which regions at the time of reading, so I left this verbiage and added a note further up about choosing locations.